client/src/components/HandMarkedPaperBallot.tsx

```diff
@@ -129,6 +131,11 @@ const ballotMetadata = ({
     ['l2', secondaryLocaleCode],
     ['p', `${pageNumber}-${pageCount}`],
   ])
+
+  if (ballotId) {
+    params.append('bid', ballotId)
```

Comment:
Because this is a ballot, it seems that `id` is more clear? And "bid" being pronounceable, seems odd. And also keeping the keys short is also good. Is there any benefit to using "bid" that I'm missing?
```suggestion
    params.append('id', ballotId)
```

Reply:
I was trying to match `bs` --> ballot style, `bid` --> ballot id. I can go either way.

Reply:
I hear ya. I really tried to figure out a way to call "ballot style" just "style" but it's never called "style" it is "ballot style".

Using `id` in this context is very clear as this really is the unique ID for the ballot.